src/compiler/operators.rs

```diff
@@ -213,6 +273,36 @@ pub fn comparison_binop_opcode(
                 }
             }
         },
+        ComparisonBinOp::LessThan => match (left, right) {
+            (RegisterOrConstant::Register(left), RegisterOrConstant::Register(right)) => {
+                OpCode::LessRR {
+                    skip_if: !skip_if,
```

Comment:
you're inverting skip_iff here when you don't want to

Reply:
Thanks!